source/administration/mmctl-cli-tool.rst

```diff
@@ -0,0 +1,2318 @@
+
+**Options**
+
+.. code-block:: sh
+
+  -a, --access-token string   Access token to use instead of username/password
+  -h, --help                  help for login
+  -m, --mfa-token string      MFA token for the credentials
+  -n, --name string           Name for the credentials
+  --no-activate               If present, it won't activate the credentials after login
+  -p, --password string       Password for the credentials
+  -u, --username string       Username for the credentials
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+   --format string the format of the command output [plain, json] (default "plain")
+
+mmctl auth renew
+^^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Renew the credentials for a given server.
+
+**Format**
+
+.. code-block:: sh
+
+   mmctl auth renew [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+   auth renew local-server
+
+**Options**
+
+.. code-block:: sh
+
+  -a, --access-token string   Access token to use instead of username/password
+  -h, --help                  help for renew
+  -m, --mfa-token string      MFA token for the credentials
+  -p, --password string       Password for the credentials
+
+Options Inherited from Parent Commands
+
+.. code-block:: sh
+
+   --format string the format of the command output [plain, json] (default "plain")
+
+mmctl auth set
+^^^^^^^^^^^^^^^^^
+
+**Description**
+
+  Set credentials to use in the following commands.
+
+**Format**
+
+.. code-block:: sh
+
+   mmctl auth set [server name] [flags]
+
+**Examples**
+
+.. code-block:: sh
+
+   auth set local-server
+
+**Options**
+
+.. code-block:: sh
+
+   -h, --help   help for set
+
+**Options Inherited from Parent Commands**
+
+.. code-block:: sh
+
+   --format string the format of the command output [plain, json] (default "plain")
+
+
+Authenticate to a server (e.g. >mmctl auth login https://test.mattermost.com). Then username and password
+(and MFA token if MFA is enabled on the account).
+
+Password
+
+.. code-block:: sh
+
+     $ mmctl auth login https://community.mattermost.com --name community --username my-username --password mysupersecret
+
+The ``login`` command can also work interactively, so if you leave any required flag empty, ``mmctl`` will ask you for it interactively:
+
+.. code-block:: sh
+
+    $ mmctl auth login https://community.mattermost.com
+    Connection name: community
+    Username: my-username
+    Password:
+
+MFA
+
+To log in with MFA, use the ``--mfa-token`` flag:
+
+.. code-block:: sh
+
+   $ mmctl auth login https://community.mattermost.com --name community --username my-username --password mysupersecret --mfa-token 123456
+
+Access Tokens
+^^^^^^^^^^^^^
+
+You can generate and use a personal access token to authenticate with a server, instead of using username and password to log in:
+
+.. code-block:: sh
+
+   $ mmctl auth login https://community.mattermost.com --name community --access-token MY_ACCESS_TOKEN
+
+
+into a Mattermost instance:
```

Comment:
Could be intentional but this sentence seems incomplete.

Reply:
Fixed.